docs/tutorial/graph_api.rst

```diff
@@ -0,0 +1,598 @@
+  e. Inverse FFT (return to spatial domain)
+  f. Normalization (account for unnormalized FFT)
+3. **Reconstruct** the 3D volume using the Feldkamp-Davis-Kress (FDK) algorithm [FeDK84]_
+
+**Why HIP graphs?** CT scanners process hundreds of projections per scan. By capturing this fixed workflow as a graph,
+you will reduce the amount of API calls required for launching the workflow on a GPU to 1 per projection, thus reducing
+launch overhead and dispatch latency to near-zero.
+
+What you will learn
+-------------------
+
+After completing this tutorial, you will be able to:
+
+* Convert a stream-based HIP application to use graphs via stream capturing
+* Create graphs manually for fine-grained control
+* Integrate graph-safe libraries like hipFFT into your graphs
+* Understand when graphs provide performance benefits
+* Apply graph concepts to your own workflows
+
+Before you begin
+----------------
+
+Required knowledge
+^^^^^^^^^^^^^^^^^^
+
+You should be comfortable writing and debugging HIP kernels, understand basic GPU memory management concepts like
+device allocation and host-to-device transfers, be familiar with HIP streams and events, and have experience using
+CMake to build C++ projects. This tutorial assumes you have written at least a few HIP programs before and understand
+concepts like grid dimensions and thread blocks.
+
+Hardware and software requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Your system needs ROCm 6.2 or later with the hipFFT library installed. The tutorial works on
+all :doc:`supported AMD GPUs <rocm-install-on-linux:reference/system-requirements>`, though at least 4 GiB of GPU
+memory are recommended for comfortable performance with the reconstruction workload. You will also need
+`git <https://git-scm.com/>`__ to check out the code repository, `CMake <https://www.cmake.org>`__ 3.21 or later to
+build the code, along with a CMake generator that supports the HIP language such as GNU Make or Ninja.
+
+.. note:: Visual Studio generators currently do not support HIP.
+
+To save the output volume, you need a recent version of `libTIFF <https://libtiff.gitlab.io/libtiff/>`__. If CMake
+cannot find libTIFF on your system, it automatically downloads and builds it.
+
+To view both the input projections and the output volume produced by this tutorial, install a scientific image viewer
+that can display 16-bit and 32-bit grayscale data, such as `Fiji <https://imagej.net/software/fiji/downloads>`__.
+Standard image viewers may be unable to correctly display the output.
+
+Optional knowledge
+^^^^^^^^^^^^^^^^^^
+
+While not required, familiarity with Fast Fourier Transform (FFT) operations will help you understand the filtering
+steps. Similarly, knowledge of medical imaging or CT reconstruction is helpful for understanding the application
+context. If you have worked with signal processing or image filtering before, you will recognize some of the applied
+concepts.
```

Comment:
I would mention in a note:

If someone isn’t interested in understanding the reconstruction algorithm, they should focus only on the main files to explore the different stream and graph solutions.